modules/standard/List.chpl

```diff
@@ -1778,7 +1778,7 @@ module List {
     :return: `true` if the contents of two lists are equal.
     :rtype: `bool`
   */
-  proc ==(a: list(?t, ?), b: list(t, ?)): bool {
+  operator ==(a: list(?t, ?), b: list(t, ?)): bool {
```

Comment:
What's the reason for having `==` for list be a standalone while `==` for map is an operator method?

Reply:
Thanks, I think I just missed it.

Reply:
I had a question about the DateTime operators, but the rest of what my name is on looks good.